modin/pandas/series.py

```diff
@@ -321,7 +321,11 @@ def __repr__(self):
             len_str = "Length: {}, ".format(len(self.index))
         else:
             len_str = ""
-        dtype_str = "dtype: {}".format(temp_str.rsplit("dtype: ", 1)[-1])
+        dtype_str = "dtype: {}".format(
+            str(self.dtype) + ")"
```

Comment:
Why is there a closing parenthesis?

Reply:
The closing parenthesis is typically taken from the `str(temp_df)`. Since I'm not using that here (see below for why), I need to provide it myself.

Reply:
Basically if we want to match Pandas, that's what we have to do. It's... a little strange honestly, but for some reason Pandas is inconsistent in repr() output:

```
In [15]: repr(pandas.Series([1]))              
Out[15]: '0    1\ndtype: int64'

In [16]: repr(pandas.Series())                 
DeprecationWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.
Out[16]: 'Series([], dtype: float64)'
```